### ISSUE TITLE
Support for Service Connector for App Service and Spring Cloud

### ISSUE THREAD

**Issue by le-neumann** [hashicorp/terraform-provider-azurerm#14187]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Microsoft recently published a preview of [Service Connector](https://docs.microsoft.com/en-us/azure/service-connector/overview). The feature is an immensely useful way of connecting storage accounts and databases to Azure App Service and Azure Spring Cloud. 

The connections can currently be created and managed through [App Service CLI](https://docs.microsoft.com/en-us/azure/service-connector/quickstart-cli-app-service-connection) and [Spring Cloud CLI](https://docs.microsoft.com/en-us/azure/service-connector/quickstart-cli-spring-cloud-connection).

### New or Affected Resource(s)

* azurerm_app_service_connection
* azurerm_spring_cloud_connection

### Potential Terraform Configuration

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

**Comment by almir0** [comment]
Should be added to Azure Container Apps as well, new resource `azurerm_container_app_service_connection`

**Comment by catriona-m** [comment]
Hi @le-neumann, thanks for raising this issue. It looks like this is now resolved by #16907 and this can now be closed. Thanks!

**Comment by Yukititit** [comment]
Hi, is the service connector of container app still have not been supported yet?
